versioned_docs/version-2.0.0/quickstart/go-gin-redis.md

```diff
@@ -0,0 +1,638 @@
+            - origin: server
+            message:
+                - type: string
+                data: "+PONG\r\n"
+    ---
+    version: api.keploy.io/v1beta2
+    kind: Generic
+    name: mocks
+    spec:
+        metadata: {}
+        genericrequests:
+            - origin: client
+            message:
+                - type: string
+                data: "*5\r\n$3\r\nset\r\n$19\r\nsomething@gmail.com\r\n$38\r\n{\"otp\":5486,\"username\":\"shivamsourav\"}\r\n$2\r\nex\r\n$5\r\n14400\r\n"
+        genericresponses:
+            - origin: server
+            message:
+                - type: string
+                data: "+OK\r\n"
+    ---
+    version: api.keploy.io/v1beta2
+    kind: Generic
+    name: mocks
+    spec:
+        metadata: {}
+        genericrequests:
+            - origin: client
+            message:
+                - type: string
+                data: "*2\r\n$3\r\nget\r\n$19\r\nsomething@gmail.com\r\n"
+        genericresponses:
+            - origin: server
+            message:
+                - type: string
+                data: "$38\r\n{\"otp\":5486,\"username\":\"shivamsourav\"}\r\n"
+  ```
+
+  Want to see if everything works as expected?
+
+  #### Run Tests
+
+  Time to put things to the test 🧪
+  
+  ```shell
+  sudo -E keploy test -c "./gin-redis" --delay 10
+  ```
+
+  > The `--delay` flag? Oh, that's just giving your app a little breather (in seconds) before the test cases come knocking.
+
+  Final thoughts? Dive deeper! Try different API calls, tweak the DB response in the `mocks.yml`, or fiddle with the request or response in `test-x.yml`. Run the tests again and see the magic unfold!✨👩‍💻👨‍💻✨
+
+  ## Wrapping it up 🎉
+
+  Congrats on the journey so far! You've seen Keploy's power, flexed your coding muscles, and had a bit of fun too! Now, go out there and keep exploring, innovating, and creating! Remember, with the right tools and a sprinkle of fun, anything's possible. 😊🚀
+
+  Happy coding! ✨👩‍💻👨‍💻✨
+  </details>
+
+  </details>
+
+  <br/>
+
+- <details> 
+  <summary><img src="/docs/img/os/macos.png" alt="MacOS" width="3%" /> MacOs </summary>
+
+  Dive straight in, but first, give **Colima** a gentle nudge with (`colima start`). Let's make sure it's awake and ready for action!
+
+  #### Add alias for Keploy 🐰:
+
+  For the sake of convenience (and a bit of Mac magic 🪄), let's set up a shortcut for Keploy:
+
+  ```bash
+  alias keploy='sudo docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v "$(pwd)":/files -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock --rm ghcr.io/keploy/keploy'
+  ```
+
+  ### Lights, Camera, Record! 🎥
+
+  Capture the test-cases-
+  ```shell
+  keploy record -c "docker run -p 3001:3001 --name RediApp --network <networkName> --name ginRedisApp gin-app:1.0"
+  ```
+
+ 🔥**Make some API calls**. Postman, Hoppscotch or even curl - take your pick!
```

Comment:
This is showing out the details and summary part

Reply:
Correction made!!